sky/clouds/service_catalog/gcp_catalog.py

```diff
@@ -27,6 +27,9 @@
 _image_df = common.read_catalog('gcp/images.csv',
                                 pull_frequency_hours=_PULL_FREQUENCY_HOURS)
 
+_quotas_df = common.read_catalog('gcp/accelerator_quota_mapping.csv',
+                                 pull_frequency_hours=_PULL_FREQUENCY_HOURS)
```

Comment:
Since `accelerator_quota_mapping.csv` is static and is not updated automatically, maybe let's not use `pull_frequency_hours`? That way we reduce unnecessary checks

```suggestion
_quotas_df = common.read_catalog('gcp/accelerator_quota_mapping.csv')
```

Reply:
Also realized the same is true for AWS - perhaps let's change that too :)